### PR TITLE
cmd/devp2p: submit Route53 changes in batches

### PR DESCRIPTION
**Pull request by fjl**
This works around the 32k RDATA character limit per change request.

### CODE REVIEW

**Comment by karalabe** [comment]
Code LGTM. I do have a question around atomicity. Previously we didn't care about the order of the changes because we just applied everything or nothing. With the chunks introduced in this PR, we need to ensure that failures don't leave us in an inconsistent state.

I'm not sure how the rest of the code works currently, but I guess the first entries in the batches should be adding new nodes, then the switchover to the root, then the deletions. This way a crash at any point will still keep things consistent. Is the rest of the route53 code structured to order it like this?

**Comment by fjl** [comment]
Yes, the changes should be ordered like that, but there is no explicit sorting. I will add that.

**Comment by fjl** [comment]
Maybe it is also time to add a test, this code is getting complicated...

**Comment by fjl** [comment]
Turns out the changes were not sorted. Now they are, and we have a test.
@karalabe PTAL.

File: cmd/devp2p/dns_route53_test.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
+package main
```
Comment (karalabe) [inline cmd/devp2p/dns_route53_test.go]: Pls add the copyright header

**Comment by fjl** [comment]
Added copyright header and one more bug fix.

**Comment by fjl** [comment]
And one more bug fix.

**Pull request merged**